source/features/bugs-tab.tsx

```diff
@@ -158,7 +166,7 @@ void features.add(__filebasename, {
 	init,
 }, {
 	include: [
-		pageDetect.isIssue,
+		pageDetect.isRepo,
```

Comment:
This changes `highlightBugsTabOnIssuePage` from being just for single issues to acting on every page. This was not necessary before so this is likely currently duplicating existing functionality

Reply:
The problem is that we don't highlight the "Bugs" label again when navigating through `isRepo` pages, because `init` only runs once. I tried to make it work with `deduplicate` (see first commit of this PR) but it doesn't work correctly. Hence I think we should just run `highlightBugsTabOnIssuePage` all the time.

Reply:
> Hence I think we should just run `highlightBugsTabOnIssuePage` all the time.

Yeah that's fine, but we probably need to drop/adapt the code that took care of this logic in the main `init` function so that the functionality does not overlap.

Maybe that was already taken care of though?

```diff
- 	issuesTab.removeAttribute('data-selected-links');
```